plugins/inputs/opentelemetry_listener/writer.go

```diff
@@ -0,0 +1,22 @@
+package opentelemetry_listener
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+)
+
+type writer struct {
+	accumulator telegraf.Accumulator
+}
```

Comment:
How about
```suggestion
type writer telegraf.Accumulator
```

Reply:
`telegraf.Accumulator` is an interface, so can't be a receiver. Nice idea, though.